flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/shuffle/KafkaShuffleExactlyOnceITCase.java

```diff
@@ -39,7 +37,7 @@
 /** Failure Recovery IT Test for KafkaShuffle. */
 public class KafkaShuffleExactlyOnceITCase extends KafkaShuffleTestBase {
 
-    @Rule public final Timeout timeout = Timeout.millis(600000L);
+    //    @RegisterExtension public final Timeout timeout = Timeout.millis(600000L);
```

Comment:
Well to have same behavior for junit5 timeout should be propagated to all the tests

Reply:
fixed it